apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/(marketplace)/components/list-button.tsx

```diff
@@ -40,14 +46,35 @@ export const CreateListingButton: React.FC<CreateListingButtonProps> = ({
         </SheetTrigger>
         <SheetContent className="w-full overflow-y-auto sm:min-w-[540px] lg:min-w-[700px]">
           <SheetHeader>
-            <SheetTitle className="mb-5 text-left">{createText}</SheetTitle>
+            <SheetTitle className="text-left">{createText}</SheetTitle>
           </SheetHeader>
-          <CreateListingsForm
-            contract={contract}
-            type={type}
-            actionText={createText}
-            setOpen={setOpen}
+          <TabButtons
```

Comment:
The tabs should be conditionally rendered.  If the chain is not supported by the NFT API - user can only enter the details manually. In that case - no tabs should be shown

Reply:
yep makes sense

Reply:
@MananTank actually i just recall our conversation last week or so: We cannot rely on "supported chains" because there is a customer whose their collection is on a supported chain, but it still doesn't work because of some parallel-chain quirky things (Sei network)

So we need to always display these tabs

Reply:
No - its the opposite - even if the chain is supported in NFT API, we show the tabs to always allow manual input - but if the chain is not supported in NFT API - both the tabs should show the same form - entering the details manually and that's why we can't show tabs in that case

Reply:
Ah yes yes

Reply:
updated